src/apps/integrations/tests/test_routes.py

```diff
@@ -0,0 +1,53 @@
+from apps.integrations.errors import UniqueIntegrationError
```

Comment:
In the project unit tests are stored under "tests/unit" subfolder, integration under "tests". Please follow this pattern (see e.g. apps/activities/tests)